CopilotKit/packages/react-core/src/hooks/use-copilot-error.ts

```
@@ -0,0 +1,185 @@
+        } catch (error) {
+          await handleError(error);
+          // Return void to indicate error was handled
+        }
+      };
+    },
+    [handleError],
+  );
+
+  const wrapSync = useCallback(
+    <T extends any[], R>(fn: (...args: T) => R) => {
+      return (...args: T): R | void => {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)